src/request/export/index.html

```diff
@@ -188,44 +199,42 @@ <h1 data-i18n="export-file-intro-heading" class="nq-h1">Save your account</h1>
 
                 <div class="page-body nq-card-body">
                     <p class="nq-light-blue nq-text flex-grow" data-i18n="export-file-intro-blue-text">
-                        Your Login File grants access to your account.
-                        Only you can access it. Download and save it.
+                        Your Login File grants access to your account. Download and save it.
                     </p>
                     <ul class="nq-list">
                         <li>
                             <svg class="nq-icon">
                                 <use xlink:href="../../../node_modules/@nimiq/style/nimiq-style.icons.svg#nq-browser-login"/>
                             </svg>
-                            <span>
-                                Your Login File is stored in your browser.<br/>
-                                Deleting it logs you out of your Account.
+                            <span data-i18n="export-file-intro-browser">
+                                Your Account is stored in your browser.
                             </span>
                         </li>
                         <li>
                             <svg class="nq-icon">
                                 <use xlink:href="../../../node_modules/@nimiq/style/nimiq-style.icons.svg#nq-skull"/>
                             </svg>
-                            <span>
-                                It is part of your browser’s storage and<br/>
-                                might be deleted by accident.
+                            <span data-i18n="export-file-intro-accident">
+                                As a part of your browser’s data it might be deleted by accident.
```

Comment:
Non-standard apostrophe.
```suggestion
                                As a part of your browser's data it might be deleted by accident.
```